admin/commands/management/add_mysql.go

```diff
@@ -143,16 +143,35 @@ func (cmd *addMySQLCommand) GetSocket() string {
 	return cmd.Socket
 }
 
+// GetCredentials figures out the credentials to be taken from CLI, credentials-source or defaults-file
 func (cmd *addMySQLCommand) GetCredentials() error {
-	creds, err := commands.ReadFromSource(cmd.CredentialsSource)
-	if err != nil {
-		return fmt.Errorf("%w", err)
+	// passed username has higher priority
```

Comment:
nit: it is not getting credential, it is mutating (if needed creds). shall we rename it?

Reply:
yep, just reused existing method

Reply:
I understand